internal/cmd/invoice_flag.go

```diff
@@ -0,0 +1,9 @@
+package cmd
+
+import "github.com/spf13/cobra"
+
+var invoiceType string
+
+func AddInvoiceType(cmd *cobra.Command) {
```

Comment:
Please add new flags on the `flags` package. You can use the [timeline](https://github.com/tursodatabase/turso-cli/blob/main/internal/flags/timeline.go#L0-L1) flag as reference. It has autocomplete and validation built in.

Reply:
thanks for this pointer, very useful!